api/cedar_metadata_templates/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import re_path
```

Comment:
A slightly better one: `from django.urls import re_path`

Similar in the `api.base.urls.py`: `from django.urls import include, re_path`